examples/todo-list/src/models/todo-list-image.model.ts

```diff
@@ -15,7 +15,7 @@ export class TodoListImage extends Entity {
   id: number;
 
   @belongsTo(() => TodoList)
-  todoListId?: number;
+  todoListId: number;
```

Comment:
+1, the foreign key is usually required (`NOT NULL` in SQL).

Reply:
I'll revert this change.

Reply:
I might misunderstand something...
> the foreign key is usually required (NOT NULL in SQL).

Doesn't it mean the change is good 🙈

Reply:
Yes, in my comment, I meant that I agreed with this proposed change too.

Please don't revert this change!

Reply:
Sorry for the confusion I created 🙈